examples/non_blocking_iid_example.py

```diff
@@ -0,0 +1,110 @@
+from time import sleep
+from decanter_ai_sdk.non_blocking_client import Client
```

Comment:
The name of the client should be different for blocking and non blocking. I think `Client` and `NonBlockingClient` is ok